cli/src/main/java/org/apache/iotdb/tool/ExportCsv.java

```diff
@@ -149,6 +153,8 @@ public static void main(String[] args) {
       System.out.println("Invalid args: " + e.getMessage());
     } catch (IoTDBConnectionException | StatementExecutionException e) {
       System.out.println("Connect failed because " + e.getMessage());
+    } catch (TException e) {
+      e.printStackTrace();
```

Comment:
Use the System.out to print the message as the others?

Reply:
ok, got this